config/routes.rb

```diff
@@ -10,7 +10,11 @@
   #ENC requests goes here
   get "node/:name" => 'hosts#externalNodes', :constraints => { :name => /[^\.][\w\.-]+/ }
 
-  resources :reports, :only => [:index, :show, :destroy] do
+  # Redirect reports with deprecation warning
+  get "/reports", :to => redirect { |params, request| DeprecationRedirect.new("config_reports", params, request.query_string, '1.12', 'Reports are renamed to Config Reports').redirect_url }
```

Comment:
oops, somehow I missed this, I removed DeprecationRedirect since I didn't see any usage... 

anyway I don't think we need to care about redirections, I don't expect someone would bookmark specific report -> https://github.com/shlomizadok/foreman/pull/5 (feel free to close if you disagree)